astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -404,14 +404,21 @@ def set_axislabel(self, text, minpad=1, **kwargs):
             The axis label text.
         minpad : float, optional
             The padding for the label in terms of axis label font size.
+        fontdict : dict, optional
+            Dictionary of settings to pass to the label
         kwargs
             Keywords are passed to :class:`matplotlib.text.Text`. These
             can include keywords to set the ``color``, ``size``, ``weight``, and
             other text properties.
         """
         self.axislabels.set_text(text)
-        self.axislabels.set_minpad(minpad)
+        # NOTE: When using plt.xlabel/plt.ylabel, minpad can get set explicitly
+        # to None so we need to make sure that in that case we change to a
+        # default numerical value, which we do in the line below.
+        self.axislabels.set_minpad(minpad or 1)
```

Comment:
This prevents one from using `minpad=0`; is that intended? For clarity, maybe `minpad if minpad is not None else 1`. Or just:
```
if minpad is None:
    minpad = 1
```

Reply:
Done